package.json

```diff
@@ -33,41 +33,32 @@
     "test": "./build.sh --test && mt dist-for-testing --quiet"
   },
   "dependencies": {
-    "@codemod-utils/ast-javascript": "^1.2.9",
-    "@codemod-utils/files": "^2.0.5",
-    "@codemod-utils/json": "^1.1.9",
+    "@codemod-utils/ast-javascript": "^1.2.13",
+    "@codemod-utils/files": "^2.0.8",
+    "@codemod-utils/json": "^1.1.14",
     "yargs": "^17.7.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.26.0",
-    "@changesets/cli": "^2.27.10",
+    "@changesets/cli": "^2.28.0",
     "@changesets/get-github-info": "^0.6.0",
-    "@codemod-utils/tests": "^1.1.8",
+    "@codemod-utils/tests": "^1.1.11",
+    "@ijlee2-frontend-configs/eslint-config-node": "^0.2.1",
+    "@ijlee2-frontend-configs/prettier": "^0.2.0",
+    "@ijlee2-frontend-configs/typescript": "^0.3.0",
     "@sondr3/minitest": "^0.1.2",
-    "@tsconfig/node18": "^18.2.4",
-    "@tsconfig/strictest": "^2.0.5",
-    "@types/node": "^18.19.67",
+    "@types/node": "^18.19.76",
     "@types/yargs": "^17.0.33",
-    "@typescript-eslint/eslint-plugin": "^8.16.0",
-    "@typescript-eslint/parser": "^8.16.0",
     "concurrently": "^9.1.0",
-    "eslint": "^8.57.1",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-import-resolver-typescript": "^3.6.3",
-    "eslint-plugin-import": "^2.31.0",
-    "eslint-plugin-n": "^17.14.0",
-    "eslint-plugin-prettier": "^5.2.1",
-    "eslint-plugin-simple-import-sort": "^12.1.1",
-    "eslint-plugin-typescript-sort-keys": "^3.3.0",
-    "prettier": "^3.4.1",
-    "typescript": "^5.7.2"
+    "eslint": "^9.20.1",
+    "prettier": "^3.5.1",
+    "typescript": "^5.7.3"
   },
   "engines": {
     "node": "18.* || >= 20"
   },
   "pnpm": {
     "overrides": {
-      "eslint-plugin-import@2.31.0>tsconfig-paths": "^4.2.0"
+      "get-tsconfig": "4.7.3"
```

Comment:
Due to a regression in `v4.7.4` that occurs when `@ijlee2-frontend-configs/eslint-config-node` and `@ijlee2-frontend-configs/typescript` are both used. The bug is present as of `v4.10.0`.

```sh
Resolve error: File '@tsconfig/strictest/tsconfig' not found.
  at resolveExtends
  at _parseTsconfig
```